dist/milo.bundle.js

```diff
@@ -2364,8 +2367,18 @@ function Component$setState(state) {
     if (state.facetsStates)
         _.eachKey(state.facetsStates, function(fctState, fctName) {
             var facet = this[fctName];
-            if (facet && typeof facet.setState == 'function')
+
+            if (!facet) {
+                try {
```

Comment:
Is there something about try/catch in the code having a notable effect on performance? Something to do with the JS engine not being able to optimise that function? If so, we should probably wrap the try/catch in a separate function.